src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -45,6 +45,7 @@
     <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" ExcludeAssets="runtime" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" ExcludeAssets="runtime" GeneratePathProperty="true" />
+    <PackageReference Include="System.Security.Cryptography.Xml" ExcludeAssets="Runtime" GeneratePathProperty="true"/>
```

Comment:
This package's package path property isn't being used, so you don't need `GeneratePathProperty`.

Similarly to another file, I'm not sure the ExcludeAssets is needed, whereas I think it is needed for the MSBuild packages, which is why they have them.